### PR TITLE
Fix issues for dontet

### PR DESCRIPTION
**Pull request by JINSEONGHUN**
### Bugs Fixed ###

Fix markdown conversion errors in dotnet

 - Issue #289

### CODE REVIEW

File: docs/application/dotnet/guides/messaging/push.md [hunk 1]
```diff
@@ -284,7 +284,8 @@ To register with the push server:
     ```
 
     > **Note**   
-	> The `PushServiceUnregister()` method is not used if the application is intended to receive push notifications continuously while it is installed on the device. When the application is uninstalled, the push service detects the event and deregisters the application automatically.
+	> The `PushServiceUnregister()` method is not used if the application is intended to receive push notifications continuously while it is installed on the device. When the application is uninstalled, the push service detects the event and deregisters the application automatically.
```
Comment (journey2w) [inline docs/application/dotnet/guides/messaging/push.md]: Please see
https://github.com/JINSEONGHUN/tizen-docs/blob/dfae7683b90ea8e04951cb0de53cc8b5ee3cea27/docs/application/dotnet/guides/messaging/push.md.

Note tag have to be indented. It means that the number of spaces in front of the sentences in Note have to be same.

File: docs/application/dotnet/guides/multimedia/camera.md [hunk 2]
```diff
@@ -50,10 +50,10 @@ The main features of the `Tizen.Multimedia.Camera` class include:
     Since devices can have multiple camera sensors with different capabilities, create a `Tizen.Multimedia.Camera` instance with a proper [Tizen.Multimedia.CameraDevice](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Multimedia.CameraDevice.html) enumeration value, determining which camera sensor is used. Usually the primary sensor is located on the back side and the secondary sensor on the front side of the device. Once the camera sensor is selected, the selected sensor starts working.
 
     > **Note**   
-	> Simultaneous use of multiple camera sensors is not allowed.
-    > The target device often supports more functionalities than the emulator.
+	> Simultaneous use of multiple camera sensors is not allowed.
```
Comment (journey2w) [inline docs/application/dotnet/guides/multimedia/camera.md]: Don't use tab but use spaces to align with the other sentences in Note.

File: docs/application/dotnet/guides/personal/account.md [hunk 3]
```diff
@@ -463,11 +464,11 @@ The following table lists the account properties that can be modified.
 
 | Account property    | Data type                          | Mandatory | Description                              |
 |-------------------|----------------------------------|---------|----------------------------------------|
-| User name           | `String`                           | Yes       | Identity of an account.If the display name and email ID are not set for an account, the user name is shown for the account on the Accounts screen in the Setting application. |
+| User name           | `String`                           | Yes       | Identity of an account.<br>If the display name and email ID are not set for an account, the user name is shown for the account on the Accounts screen in the Setting application. |
```
Comment (journey2w) [inline docs/application/dotnet/guides/personal/account.md]: Remove code tag(\'  \') in Data type column. Apply all item in Data type column.

File: docs/application/dotnet/guides/personal/account.md [hunk 4]
```diff
@@ -491,7 +492,7 @@ The following table lists the properties that can be defined for each account pr
 | Icon                      | `String`  | Yes       | File path of the account provider icon.  <br>    The icon size is:<br> - 72 x 72 for Xhigh (HD) <br> - 48 x 48 for High (WVGA) <br> Since the icon is used in **Settings > Accounts**, place the icon in a shared directory. |
```
Comment (journey2w) [inline docs/application/dotnet/guides/personal/account.md]: Remove code tag(' ') in Data type column. Apply all item in Data type column.

File: docs/application/dotnet/guides/multimedia/metadata.md [hunk 5]
```diff
@@ -252,8 +252,8 @@ The metadata is available with the properties and methods of the [Tizen.Multimed
 | Conductor            | Conductor of the content               | `Metadata.Conductor`                     |
 | Unsync lyrics        | Asynchronous lyrics of the content     | `Metadata.UnsyncLyrics`                  |
 | Sync lyrics          | Synchronous lyrics of the content      | `Metadata.SyncLyricsCount`<br>`MetadataExtractor.GetSyncLyrics()`<br>`SyncLyrics.Lyrics`<br>`SyncLyrics.Timestamp` |
-| Recorded date        | Description                            | Property or method                       |
-| Duration             | Duration of the content                | `Metadata.Duration`                      |
+| Recorded date        | Description                            | `Metadata.DateRecorded`                  |
```
Comment (mijin80lee) [inline docs/application/dotnet/guides/multimedia/metadata.md]: line 255: Fill in the right value. (ref. TD Recorded date, Recorded date of the content, Metadata.DateRecorded)
line 256: Fill in the right value. (ref. TD Orientation, Orientation of the content, Metadata.Rotation)

File: docs/application/dotnet/guides/app-management/watch-app.md [hunk 6]
```diff
@@ -200,7 +200,7 @@ protected override void OnCreate()
     conformant.SetContent(_mainLayout);
 ```
 
-> **Note**
+> **Note**
```
Comment (mijin80lee) [inline docs/application/dotnet/guides/app-management/watch-app.md]: line 203: Note form was not applied.

File: docs/application/dotnet/guides/connectivity/telephony.md [hunk 7]
```diff
@@ -182,7 +182,6 @@ To retrieve information from a SIM card and monitor when the SIM state changes,
 
         ```
         Log.Info(Globals.LogTag, "Iccid = " + _sim.IccId);
-
      ```
```
Comment (mijin80lee) [inline docs/application/dotnet/guides/connectivity/telephony.md]: line 185: The closing '```' must be in the same position as the opening one.
Remove unnecessary line.
(you could see a blank code block in the 'View' mode.)

File: docs/application/dotnet/guides/location-sensors/location.md [hunk 8]
```diff
@@ -32,8 +32,9 @@ The [Tizen.Location](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Locat
 
 
 > **Note**
```
Comment (mijin80lee) [inline docs/application/dotnet/guides/location-sensors/location.md]: line 34: Note form was not applied.

File: docs/application/dotnet/guides/multimedia/camera.md [hunk 9]
```diff
@@ -49,11 +49,11 @@ The main features of the `Tizen.Multimedia.Camera` class include:
 
     Since devices can have multiple camera sensors with different capabilities, create a `Tizen.Multimedia.Camera` instance with a proper [Tizen.Multimedia.CameraDevice](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Multimedia.CameraDevice.html) enumeration value, determining which camera sensor is used. Usually the primary sensor is located on the back side and the secondary sensor on the front side of the device. Once the camera sensor is selected, the selected sensor starts working.
 
-    > **Note**   
-	> Simultaneous use of multiple camera sensors is not allowed.
-    > The target device often supports more functionalities than the emulator.
+  > **Note**
```
Comment (mijin80lee) [inline docs/application/dotnet/guides/multimedia/camera.md]: line 52: Note form was not applied.
line 325: Note form was not applied.

File: docs/application/dotnet/guides/multimedia/media-playback.md [hunk 10]
```diff
@@ -309,14 +309,14 @@ You set the display parameters of a video file with the `DisplaySettings` proper
...
 
-    2.  If the display is not visible, make it visible by setting the `IsVisible` property to `true`:
+    2.  If the display is not visible, make it visible by setting the `IsVisible` property to `true`:  
 
-        ​```
+        ```
         if (player.DisplaySettings.IsVisible == false)
         {
-            player.DisplaySettings.IsVisible = true;
+          player.DisplaySettings.IsVisible = true;
```
Comment (mijin80lee) [inline docs/application/dotnet/guides/multimedia/media-playback.md]: line 317: Add 2 spaces at the beginning.

File: docs/application/dotnet/guides/nui/animation-types.md [hunk 11]
```diff
@@ -119,7 +119,7 @@ animation.AnimatePath(view, path, Vector3.XAxis, 0, 5000, new AlphaFunction(Alph
 animation.Play();
 ```
 
-> **Note**
+> **Note**
```
Comment (mijin80lee) [inline docs/application/dotnet/guides/nui/animation-types.md]: line 122: Note form was not applied.

File: docs/application/dotnet/guides/personal/account.md [hunk 12]
```diff
@@ -463,22 +464,22 @@ The following table lists the account properties that can be modified.
...
-| Sync support        | `Integer`                          | No        | Current synchronization status.          |
-| Source              | `String`                           | No        | Source of an account.                    |
-| User text           | `String`                           | No        | String array, which you can use freely.  |
-| User int            | `Integer`                          | No        | Integer array, which you can use freely. |
-| Custom              | Key-value `string` pairs           | No        | Key-value pairs, which you can use freely. |
-| Safe account handle | `SafeAccountHandle`                | No        | Read-only account handle of an account.  |
+| User name           | String                           | Yes       | Identity of an account.<br>If the display name and email ID are not set for an account, the user name is shown for the account on the Accounts screen in the Setting application. |
+| Display name        | String                           | No        | Display name of an account.Display name is shown for the account on the Accounts screen in the Setting application. |
```
Comment (mijin80lee) [inline docs/application/dotnet/guides/personal/account.md]: line 468: The separated lines are merged as one line.
line 495: The separated lines are merged as one line.
(Email capability:, Photo capability:, Video capability:, Music capability:, Document capability:, Message capability:, Game capability:)

File: docs/application/dotnet/guides/text-input/tts.md [hunk 13]
```diff
@@ -540,7 +540,6 @@ To start, pause, and stop the playback:
    > **Note**
```
Comment (mijin80lee) [inline docs/application/dotnet/guides/text-input/tts.md]: line 540: Note form was not applied.
line 543,555: Need to be checked. ('```')

File: docs/application/dotnet/guides/connectivity/wifi.md [hunk 14]
```diff
@@ -19,16 +19,16 @@ The main features of the Tizen.Network.WiFi namespace include:
...
-
-            | Information         | Description                              |
-            |-------------------|----------------------------------------|
-            | Authentication type | Wi-Fi EAP phase2 authentication type     |
-            | EAP type            | Wi-Fi EAP type                           |
-            | CA certificate      | EAP CA certificate (valid only if the EAP type is TLS) |
-            | Client certificate  | EAP client certificate (valid only if the EAP type is TLS) |
-            | Passphrase          | EAP passphrase (valid if the EAP type is PEAP or TTLS) |
-            | Private key file    | EAP private key file (valid only if the EAP type is TLS) |
+        **Table: Access point EAP information**
```
Comment (mijin80lee) [inline docs/application/dotnet/guides/connectivity/wifi.md]: line 22: Table form need to be checked.

**Pull request merged**